ramls/patron.raml

```diff
@@ -192,43 +192,53 @@ traits:
                   description: The UUID of a FOLIO hold request
                   type: string
                   pattern: ^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$
-              put:
-                description: Not implemented
-                responses:
-                  501:
-                    description: Not implemented
-              delete:
-                description: Removes the specified hold request
-                responses:
-                  204:
-                    description: The specified hold request was removed
-                  400:
-                    description: Bad request
-                    body:
-                      text/plain:
-                        example: |
-                          unable to process request -- constraint violation
-                  401:
-                    description: Not authorized to perform requested action
-                    body:
-                      text/plain:
-                        example: unable to delete hold -- unauthorized
-                  404:
-                    description: Item with a given ID not found
-                    body:
-                      text/plain:
-                        example: hold not found
-                  403:
-                    description: Access Denied
-                    body:
-                      text/plain:
-                        example: Access Denied
-                  500:
-                    description: |
-                      Internal server error, e.g. due to misconfiguration
-                    body:
-                      text/plain:
-                        example: internal server error, contact administrator
+              /cancel:
```

Comment:
I'll simplify it to /patron/account/{id}/hold/{holdId}/cancel.
There are a couple of /hold methods under instances and item that one day we may want to migrate to this new path. 
This new API will take a hold-cancelation.json object that has required fields for cancelling a request. It will retrieve the Hold entity first, then performs a PUT, then modifies the Hold entity to have the new cancellation fields if successful and returns.

Reply:
Cool! I was going to suggest doing the GET first after thinking about it on the drive in to work this morning. That should simplify the API impl code quite a bit.

Reply:
I redid the API and moved the GET call in front of the PUT. Please re-check.